### PR TITLE
Native Ledger Support for Dashboard

### PR DESCRIPTION
**Pull request by filippoweb3**
The dashboard will soon roll out native support for Ledger 

See Ross demo here:  https://www.loom.com/share/7fc4aeade10243f9a92c40a818fdd292

### CODE REVIEW

**Comment by filippoweb3** [comment]
@rossbulat here the additions for the native Ladger support

**Comment by rossbulat** [comment]
Thanks @filippoweb3, if we can have a walkthrough on the wiki by the final Ledger release - I imagine the beta period will last 1 - 2 months on our monthly rollout - we'll be positioned very nicely indeed for Stax support.

**Comment by filippoweb3** [comment]
@rossbulat I noticed that the native ledger support has been released as an experimental feature. I tried to use it and it works although it does not fetch the account names, so if you have multiple accounts it's messy. This PR is a minor addition to the staking page, IMO it can be merged but let me know your thoughts!

File: docs/learn/learn-staking.md [hunk 1]
```diff
@@ -16,7 +16,8 @@ Start your staking journey or explore more information about staking on
 [Staking Dashboard](https://staking.polkadot.network/#/overview) that makes staking much easier and
 check this
 [extensive article list](https://support.polkadot.network/support/solutions/articles/65000182104) to
-help you get started.
+help you get started. The dashboard has native [Ledger](../general/ledger.md) support, meaning that
+you do not need an extension or a wallet to use it, just your ledger device.
```
Comment (profradha) [inline docs/learn/learn-staking.md]:
```suggestion
help you get started. The dashboard supports [Ledger](../general/ledger.md) devices natively and does not require an extension or wallet as an interface.
```

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**